src/main/src/main/java/org/geoserver/config/GeoServerPluginConfigurator.java

```diff
@@ -0,0 +1,32 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
The header should only say "2016 Open Source Geospatial Foundation - all rights reserved", without Openplans below